QuantumEspresso/qe66_archer2_gnu.md

```diff
@@ -4,23 +4,35 @@ Build Instructions for QE 6.6 on ARCHER2
 For version 6.6 on ARCHER using the GCC compilers
 
 ```bash
-module restore PrgEnv-gnu
-module load cray-libsci
-module load cray-fftw
 wget https://github.com/QEF/q-e/archive/qe-6.6.tar.gz
 tar -xvf qe-6.6.tar.gz
 cd q-e-qe-6.6
+
+module restore PrgEnv-gnu
+module load cray-libsci cray-fftw cray-hdf5-parallel
+
 export CC=cc
 export FC=ftn
 export F77=ftn
 export F90=ftn
-./configure --prefix=${PATH_TO}/q-e-qe-6.6
+
+export BLAS_LIBS=/opt/cray/pe/libsci/20.10.1.2/gnu/9.1/x86_64/lib/libsci_gnu_mpi.a
+export LAPACK_LIBS=/opt/cray/pe/libsci/20.10.1.2/gnu/9.1/x86_64/lib/libsci_gnu_mpi.a
+export SCALAPACK_LIBS=/opt/cray/pe/libsci/20.10.1.2/gnu/9.1/x86_64/lib/libsci_gnu_mpi.a
+export FFT_LIBS=/opt/cray/pe/fftw/3.3.8.8/x86_rome/lib/libfftw3_mpi.a
```

Comment:
Should these not be empty if you are using the compiler wrapper scripts as they include the libraries by default? i.e.:

```
export BLAS_LIBS=
export LAPACK_LIBS=
export SCALAPACK_LIBS=
export FFT_LIBS=
```

If the build tool does not like them being completely empty, you can set them all to a single space " ". Or, is there something in the QE build that means that these have to be explicitly specified?

Reply:
I hadn't thought of that but have now tested and setting them all to a single space works (if empty, QE will try to use its own version of these and things go bad). I've changed the instructions